pkg/remote/github/repository.go

```diff
@@ -159,3 +170,56 @@ func (r githubRepository) ListTeams() ([]int, error) {
 	}
 	return results, nil
 }
+
+type listTeamMembershipsQuery struct {
+	Organization struct {
+		Team struct {
+			Members struct {
+				Nodes []struct {
+					Login string
+				}
+				PageInfo struct {
+					EndCursor   githubv4.String
+					HasNextPage bool
+				}
+			} `graphql:"members(first: 100, after: $cursor)"`
+		} `graphql:"team(slug: $slug)"`
+	} `graphql:"organization(login: $login)"`
+}
+
+func (r githubRepository) ListTeamMemberships() ([]string, error) {
+	teamList, err := r.ListTeams()
+	if err != nil {
+		return nil, err
+	}
+
+	query := listTeamMembershipsQuery{}
+	results := make([]string, 0)
+	if r.config.Organization == "" {
+		return results, nil
+	}
+	variables := map[string]interface{}{
+		"cursor": (*githubv4.String)(nil),
+		"slug":   (githubv4.String)(""),
+		"login":  (githubv4.String)(r.config.Organization),
+	}
+
+	for _, team := range teamList {
+		variables["slug"] = (githubv4.String)(team.Slug)
+		variables["cursor"] = (*githubv4.String)(nil)
+		for {
+			err := r.client.Query(r.ctx, &query, variables)
+			if err != nil {
+				return nil, err
+			}
+			for _, membership := range query.Organization.Team.Members.Nodes {
+				results = append(results, fmt.Sprintf("%d:%s", team.DatabaseId, membership.Login))
+			}
+			if !query.Organization.Team.Members.PageInfo.HasNextPage {
+				break
+			}
+			variables["cursor"] = githubv4.NewString(query.Organization.Team.Members.PageInfo.EndCursor)
```

Comment:
Can you try without `NewString` there ? It seems that we do not need a pointer for the cursor. I've used this for github_branch_protection but I didn't refactored other resources atm.

Reply:
It works indeed